x-pack/plugins/cloud_security_posture/common/utils/helpers.ts

```diff
@@ -5,6 +5,7 @@
  * 2.0.
  */
 
+import { Truthy } from 'lodash';
```

Comment:
```suggestion
import type { Truthy } from 'lodash';
```
it'd be interesting to see the `webpack bundle analyzer` output of our bundle with/without this additional `type` import. 

i'm not sure what kibana optimizer is doing, but it's possible importing it like this would result in us requiring all of `lodash` bundle.